jenkins/jobs/vgpu.yaml

```diff
@@ -0,0 +1,70 @@
+- job:
+    name: 'test-vgpu'
+    node: '{node}'
+
+    wrappers:
+      - timeout:
+          timeout: 40  # Timeout in *minutes*
+          fail: true  # A job run that exceeds the timeout will cause a failure
+      - timestamps
+
+    builders:
+      #- net-info
+      - shell: |
+          #!/bin/bash -eux
+          env
+
+          export ZUUL_CHANGES=${ZUUL_CHANGES:-""}
+          export ZUUL_REF=${ZUUL_REF:-""}
+          export ZUUL_CHANGE=${ZUUL_CHANGE:-""}
+
+          echo "Testing vgpu on xenserver"
+          export VGPU_TEST_LOG_DIR=$WORKSPACE/logs
+          export OPENSTACK_LOGS=$VGPU_TEST_LOG_DIR/opentack_logs
```

Comment:
this path is different from the $LOG_PATH used in logs uploading section - line 56

Reply:
Will fix it

Reply:
And there is a typo (opentack_logs)